src/scripts/api.ts

```diff
@@ -689,7 +689,8 @@ export class ComfyApi extends EventTarget {
         Running: data.queue_running.map((prompt: Record<number, any>) => ({
           taskType: 'Running',
           prompt,
-          remove: { name: 'Cancel', cb: () => api.interrupt() }
+          // prompt[1] is the prompt id
+          remove: { name: 'Cancel', cb: () => api.interrupt(prompt[1]) }
```

Comment:
prompt[1] is not the most readable, but this bit of code is only used by the legacy UI and it's somewhat complicated to have nicer types here, so I did a low-effort thing.